crates/pgls_splinter/build.rs

```diff
@@ -107,3 +128,39 @@ fn add_not_null_markers(content: &str) -> String {
 
     result
 }
+
+fn split_queries(content: &str) -> (String, String) {
```

Comment:
as I understand it, rules are per default put into the generic category.
what happens if supabase adds a new rule to the spliter that's supabase-specific?
that might break linting in case a table isn't defined for non-supabase databases, right?
should we throw during build if there's an unknown rule in the splinter?